js/start.md

```diff
@@ -275,454 +229,570 @@ To install React-specific Amplify UI components, run the following command:
+$ ng serve
 ```
 
-If you have created your app with *expo* in previous steps, you can [**skip**](#step-3-set-up-the-app-backend) this section.
+See the [Angular Guide](https://aws-amplify.github.io/amplify-js/media/angular_guide){: target='_new'} for details and usage.
+{: .callout .callout--action}
 
-Amplify provides native libraries for React Native to support Amazon Cognito sign-in process. If you are using [Expo v25.0.0 or greater](https://blog.expo.io/expo-sdk-v25-0-0-is-now-available-714d10a8c3f7), those libraries are already included in your dependencies. Otherwise, you need to [link](https://facebook.github.io/react-native/docs/linking-libraries-ios.html) those libraries to your project.
-{: .callout .callout--info}
+</div>
+<div id="ionic" class="tab-content">
 
-Following example shows how you can link the libraries for a project that is created with *react-native init*:
+Use the Ionic CLI to bootstrap a new Ionic app:
 
 ```bash
-$ react-native init myReactNativeApp
-$ cd myReactNativeApp
-$ npm install --save aws-amplify
-$ npm install --save aws-amplify-react-native
-$ react-native link
+$ npm install -g ionic
+$ ionic start myAmplifyProject blank --type=angular 
+$ cd myAmplifyProject
 ```
 
-</div>
-<div id="angular" class="tab-content">
+Inside the app directory, install Amplify and run your app:
 
 ```bash
-$ npm install --save aws-amplify
+$ npm install --save @aws-amplify/api @aws-amplify/pubsub
+$ npm start
 ```
 
-In addition to *aws-amplify* core, you can install our angular module which provides a service provider, helpers, and components:
+In addition to `aws-amplify` core, you can install the Angular Ionic modules which provide a service provider, helpers, and components:
 
 ```bash
 $ npm install --save aws-amplify-angular
 ``` 
 
-See the [Angular Guide](https://aws-amplify.github.io/amplify-js/media/angular_guide){: target='_new'} for details and usage.
+See the [Ionic Guide](https://aws-amplify.github.io/docs/js/angular){: target='_new'} for details and usage.
 {: .callout .callout--action}
 
 </div>
-<div id="ionic" class="tab-content">
 
-```bash
-$ npm install --save aws-amplify
-```
+<div id="vue" class="tab-content">
 
-In addition to `aws-amplify` core, you can install our angular and ionic modules which provide a service provider, helpers, and components:
-
-</div>
-<div id="ionic" class="tab-content">
+Use the Vue CLI to bootstrap a new Vue app (selecting the defaults will work for this project):
 
 ```bash
-$ npm install --save aws-amplify-angular
-``` 
-
-See the [Ionic Guide](https://aws-amplify.github.io/amplify-js/media/ionic_guide){: target='_new'} for details and usage.
-{: .callout .callout--action}
+$ npm install -g @vue/cli
+$ vue create myAmplifyProject 
+$ cd myAmplifyProject
+```
 
-</div>
-
-<div id="vue" class="tab-content">
+Inside the app directory, install Amplify and run your app:
 
 ```bash
-$ npm install --save aws-amplify
+$ yarn add @aws-amplify/api @aws-amplify/pubsub
+$ yarn serve
```

Comment:
Minor: It looks like our getting started sections inconsistently use npm and yarn.